public/app/features/logs/logsModel.ts

```diff
@@ -564,17 +568,15 @@ function adjustMetaInfo(logsModel: LogsModel, visibleRangeMs?: number, requested
 
       if (canShowCoverage) {
         const coverage = ((visibleRangeMs / requestedRangeMs) * 100).toFixed(2);
```

Comment:
Tested backward/forward changing the timeline to see of it reflects the coverage, looks good 👍 

![backward](https://github.com/user-attachments/assets/5aa69f43-c1ad-418f-b710-e536e49c5fd1)

![forward](https://github.com/user-attachments/assets/f98377ff-00c5-4c18-9585-ad08c8da0c4c)